e2e/package.json

```diff
@@ -28,6 +28,7 @@
     "@types/react-dom": "^18",
     "@types/webpack": "5.28.0",
     "fast-glob": "^3.3.1",
+    "loader-utils": "^2.0.4",
```

Comment:
it is recommended to use v3

Reply:
I have tried to upgrade this before, but the api used in other sub-program has changed, which will cause the test to failed, so I haven't upgraded for the time being. I opened an issue, and I will upgrade it later: #332 .